### PR TITLE
Rebase PRs on top of target branch

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

https://github.com/thoth-station/solver/pull/3495

And other such pull requests

### CODE REVIEW

File: kebechet/managers/update/update.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 # Kebechet
-# Copyright(C) 2018, 2019, 2020 Fridolin Pokorny
+# Copyright(C) 2018, 2019, 2020, 2021 Fridolin Pokorny
 #
```
Comment (fridex) [inline kebechet/managers/update/update.py:4]:
```suggestion
# Copyright(C) 2021 Kevin Postlethwait
#
```

**Comment by KPostOffice** [comment]
@fridex PTAL

**Comment by KPostOffice** [comment]
See: https://github.com/KPostOffice/khebhut_test/pull/38

**Review by fridex — APPROVED** [review]
looks good to me 🥇 👍🏻

**Pull request merged**